esterni/verbali/verbale-esterno_2020-02-25.tex

```diff
@@ -0,0 +1,125 @@
+\setUso{Esterno}
+\setDescrizione{Verbale dell'incontro di GruppOne del 25/02/2020}
+\setModifiche{%
+\cellcolor{white!80!lightgray!100}& Alessandro Rizzo & 2020--02--27 & approva documento \\%
+\cellcolor{white!80!lightgray!100}& Alberto Cocco & 2020--02--26 & verifica verbale \\%
+\multirow{-3}{*}{0.1.0}& Alessandro Rizzo & 2020--02--25 & stendi verbale %
+}
+
+\begin{document}
+
+\input{../../commons/template}
+
+\section{Informazioni logistiche}%
+\label{sec:informazioni_logistiche}
+
+\begin{description}
+  \item [Luogo] chiamata Hangouts
+  \item [Data] 25/02/2020
+  \item [Ora] 12:30 \symbol{8594} 12:55
+\end{description}
+
+\subsection{Membri del gruppo presenti}%
+\label{sub:membri_del_gruppo_presenti}
+
+\begin{enumerate}
+  \item Alberto Cocco
+  \item Luca Ercole
+  \item Alberto Gobbo
+  \item Alessandro Rizzo
+\end{enumerate}
+% sub:membri_del_gruppo_presenti (end)
+
+\subsection{Altri partecipanti}%
+\label{sub:altri_partecipanti}
+
+\begin{enumerate}
+  \item Prof.\ Riccardo Cardin (committente)
+\end{enumerate}
+
+% sub:altri_partecipanti (end)
+% sec:informazioni_logistiche (end)
+
+\section{Introduzione}%
+\label{sec:introduzione}
+
+L'incontro, avvenuto tramite chiamata Hangouts, ci è servito per chiarire i dubbi riguardanti il giudizio ricevuto in RR e approfondire alcuni dettagli relativi al PoC.
+
+\section{Ordine del giorno}%
+\label{sec:ordine_del_giorno}
+
+\begin{itemize}
+  \item Discussione UUC 4.3
+  \item Discussione AUC 3
+  \item Discussione sul tracciamento di casi d'uso e requisiti
+  \item Struttura del PoC
+  \item Tecnologie da usare nel PoC.
+\end{itemize}
+% sec:registro_delle_decisioni (end)
+\section{Discussione UUC 4.3}%
+\label{sec:discussione_uuc_4.3}
+\textbf{Domanda:} Il motivo per il quale vediamo UUC4.3 come un'estensione di UUC4.2, è il fatto che l'utente può selezionare una o più organizzazioni e, nel momento in cui una di queste sia privata e quindi richieda l'autenticazione LDAP, allora si scateni il caso d'uso UUC4.3. Quindi se UUC4.3 non fosse un'estensione UUC4.2 ma una sua pre-condizione, l'autenticazione LDAP verrebbe effettuata a priori su tutte le organizzazioni selezionate, il che sarebbe vero solo se tutte le organizzazioni lo richiedano (sia private che pubbliche), cosa che non è richiesta da capitolato (solo private), in che modo è sbagliata questa visione del caso d'uso?
+
+\textbf{Risposta:} Bisogna innanzitutto capire come UML può essere utilizzato per modellare i casi d'uso, questi ultimi infatti definiscono solamente le funzionalità del nostro software. Il professore ha ribadito questo fatto per dire che la selezione di una organizzazione è un dettaglio implementativo non una funzionalità, la funzionalità offerta invece è il collegamento ad una organizzazione dunque l'estensione non ha motivo di esserci così come il caso d'uso relativo alla selezione.
+In caso il modo corretto di modellare un caso d'uso di questo tipo sarebbe prevedere 4.2 come precondizione di 4.3.
```

Comment:
In caso il modo corretto di modellare un caso d'uso di questo tipo sarebbe prevedere **4.3** come precondizione di **4.2**.